### PR TITLE
Fix up UI for demo

### LINKED ISSUE #67: Fix up UI for v1.2 demo

### PR DESCRIPTION
**Pull request by LimJunxue**
Also made dynamic rendering of optional task fields

### CODE REVIEW

**Review by kaixiangtay — APPROVED** [review]
LGTM, can be merged

File: src/main/java/seedu/address/ui/TaskCardDetails.java [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+     * Asserts that it is not blank because status cannot be created blank.
+     * Sets style class according to its value.
+     *
+     * @param task Task to be displayed.
+     */
+    private void setStatus(Task task) {
+        String statusValue = task.getStatus().value;
+        assert !statusValue.isBlank() : "Status cannot be displayed blank.";
+
+        boolean isStatusDone = task.getStatus().isDone();
```
Comment (maxxng) [inline src/main/java/seedu/address/ui/TaskCardDetails.java]: Can consider refactor through law of demeter?

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 2]
```diff
@@ -3,7 +3,7 @@
 import static java.util.Objects.requireNonNull;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_DEADLINE;
 import static seedu.address.logic.parser.CliSyntax.PREFIX_DESCRIPTION;
-import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
+//import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
```
Comment (maxxng) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:6]: Should we delete this instead?

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 3]
```diff
@@ -19,19 +19,19 @@
...
             + PREFIX_DESCRIPTION + "DESCRIPTION "
-            + PREFIX_TAG + "TAG]...\n"
+            + PREFIX_TAG + "TAG]...\n\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_TITLE + "This is a task "
             + PREFIX_DEADLINE + "12 Oct 2012 "
-            + PREFIX_STARTTIME + "15:30"
-            + PREFIX_EMAIL + "johnd@example.com "
+            + PREFIX_STARTTIME + "15:30 "
+            //+ PREFIX_EMAIL + "johnd@example.com "
```
Comment (maxxng) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:34]: Should we delete this instead?

**Review by maxxng — APPROVED** [review]

File: src/main/java/seedu/address/ui/TaskCardDetails.java (hunk 1 above)
Comment (LimJunxue) [reply src/main/java/seedu/address/ui/TaskCardDetails.java]: Alright thanks!

File: src/main/java/seedu/address/logic/commands/AddCommand.java (hunk 2 above)
Comment (LimJunxue) [reply src/main/java/seedu/address/logic/commands/AddCommand.java:6]: It is in ongoing works.

File: src/main/java/seedu/address/logic/commands/AddCommand.java (hunk 3 above)
Comment (LimJunxue) [reply src/main/java/seedu/address/logic/commands/AddCommand.java:34]: It is in ongoing works.

**Pull request merged**